dn

```diff
@@ -8,11 +8,22 @@ dno() {
   echo " * $2" >> ~/dn/$1
 }
 
+# edit note for a given date, e.g. dnoe 2019-11-02
+# if no date given, vim lists the ~/dn folder, then pick one to edit
```

Comment:
To make it clear what I'm talking about in my PR comment - I suggest removing this and the other command documentation here into the README.